webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/apidata/JsonResponseObjects.scala

```diff
@@ -181,137 +186,148 @@ object JsonResponseObjects {
 
   sealed trait JRTechnique {
     def id: String
-    def name : String
+
+    def name: String
+
     def version: String
+
     def source: String
   }
 
-  final case class JRBuiltInTechnique (
-      name: String
-    , id : String
+  final case class JRBuiltInTechnique(
+    name     : String
+    , id     : String
     , version: String
-  ) extends  JRTechnique  { val source = "builtin" }
+  ) extends JRTechnique {
+    val source = "builtin"
+  }
 
-  final case class JREditorTechnique (
-      name: String
-    , version: String
-    , id : String
-    , category    : String
-    , methodCalls : List[JRTechniqueElem]
-    , description : String
-    , parameters  : Seq[JRTechniqueParameter]
+  final case class JREditorTechnique(
+    name         : String
+    , version    : String
+    , id         : String
+    , category   : String
+    , methodCalls: List[JRTechniqueElem]
+    , description: String
+    , parameters : Seq[JRTechniqueParameter]
     , resources  : Seq[JRTechniqueResource]
-  ) extends  JRTechnique  { val source = "editor" }
+  ) extends JRTechnique {
+    val source = "editor"
+  }
 
-  final case class JRTechniqueParameter (
-      id          : String
-    , name        : String
-    , description : String
-    , mayBeEmpty    : Boolean
+  final case class JRTechniqueParameter(
+    id           : String
+    , name       : String
+    , description: String
+    , mayBeEmpty : Boolean
   )
+
   object JRTechniqueParameter {
-    def from (param : TechniqueParameter) = {
-      JRTechniqueParameter (
+    def from(param: TechniqueParameter) = {
+      JRTechniqueParameter(
         param.id.value
-      , param.name.value
-      , param.description
-      , param.mayBeEmpty
+        , param.name.value
+        , param.description
+        , param.mayBeEmpty
       )
     }
   }
 
-  final case class JRTechniqueResource (
-      path  : String
```

Comment:
Your IDE broke alignement of all case classes first parameter

Reply:
Damn it :weary: